ocs_ci/helpers/dr_helpers.py

```
@@ -1783,6 +1783,78 @@ def get_all_drclusters():
+    """
+    protected_apps = []
+    for workload in workloads:
+        if workload.workload_type == constants.APPLICATION_SET:
+            app_name = workload._get_applicaionset_name()
+        elif getattr(workload, "discovered_apps_placement_name", None):
+            app_name = workload.discovered_apps_placement_name
+        else:
+            raise ValueError(
+                f"Unsupported workload type for DR Topology validation: "
+                f"{workload.workload_type}"
+            )
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)